### ISSUE TITLE
Adding Example for FastApi OpenAPI documentation

### ISSUE THREAD

**Issue by julien-roy-replicant** [RobertCraigie/prisma-client-py#715]
## Problem

Currently for my pydantic models, I use the example field to document FastAPI:

` = Field(..., example="This is my example for SwaggerUI")`

## Suggested solution

Would it be possible through comment in the schema.prisma to add these examples?

## Alternatives

Open to any solution.

## Additional context

I want to completely remove my custom pydantic object definitions and rely on your generated models and partials (which I find amazing!)

**Comment by RobertCraigie** [comment]
Yeah this would be doable using triple-slash comments on your schema, e.g.

```prisma
model User {
  /// @example("my example value")
  name String
}
```

Does that sound good to you?

**Comment by julien-roy-replicant** [comment]
Yes perfect!

Could that be generic enough to also include other pydantic arguments like `min_items`, `regex`, etc ?

**Comment by RobertCraigie** [comment]
Not with the proposed syntax but maybe it should be something like this instead?

```prisma
model User {
  /// @pydantic.example("my example value") @pydantic.min_length(4)
  name String
}
```

Where the `$` in the `@pydantic.$()` format is not validated and just passed straight through to pydantic.

**Comment by julien-roy-replicant** [comment]
Yes, I really like this proposition :)
